app/views/shared/_display_ad.html.erb

```diff
@@ -32,6 +32,15 @@
             <%= I18n.t("display_ad.menu.manage_preferences") %>
           </a>
         </li>
+        <% if Page.exists?(slug: "billboards") %>
```

Comment:
Question: Why are we using this if condition to show the element? 

We use it a few lines above too, but in that case its used to ensure that the url that we redirect to exists:

```
 <% if Page.exists?(slug: "billboards") %>
          <li>
            <a class="crayons-link crayons-link--block" href='/billboards'>
              <%= crayons_icon_tag("info", aria_hidden: true, class: "c-btn__icon") %>
              <%= I18n.t("display_ad.menu.what_is_a_billboard") %>
            </a>
          </li>
        <% end %>
```

Reply:
@lboogie04 a display_ad and a billboard are synonymous terms, we've decided to use the word billboard more recently but its still being called display_ad in the code. Eventually we hope to rename it across the codebase and UI. 

`Page.exists?(slug: "billboards")` checks if the information page for billboards exists, but this is not related to whether or not we display a report link on the billboard/display_ad itself. We should still have the ability to report a billboard/display_ad even if there is no information page describing billboards. 

Does this make sense? Let me know if we need to chat more about it, i know that these terms can be a bit confusing.